lib/BridgeAbstract.php

```diff
@@ -267,6 +267,15 @@ function($i){ return $i['name']; }, // Just display parameter names
 
 	}
 
+	/**
+	 * Resets bridge internal state (items, inputs, context)
+	 */
+	public function clearDatas(){
```

Comment:
I just realized this is insufficient for clearing the bridge if collectData is used (in testBridgeItems) because each bridge can have its own private internal states. I will just reinstantiate the bridge instead.